jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -261,6 +262,39 @@ class EndpointK @JvmOverloads constructor(
         return _transceiver.isReceivingVideo()
     }
 
+    /**
+     * Update media direction of {@link ChannelShim}s associated
+     * with this Endpoint.
+     *
+     * When media direction is set to 'sendrecv' JVB will
+     * accept incoming media from endpoint and forward it to
+     * other endpoints in a conference. Other endpoint's media
+     * will also be forwarded to current endpoint.
+     * When media direction is set to 'sendonly' JVB will
+     * NOT accept incoming media from this endpoint (not yet implemented), but
+     * media from other endpoints will be forwarded to this endpoint.
+     * When media direction is set to 'recvonly' JVB will
+     * accept incoming media from this endpoint, but will not forward
+     * other endpoint's media to this endpoint.
+     * When media direction is set to 'inactive' JVB will
+     * neither accept incoming media nor forward media from other endpoints.
+     *
+     * @param type media type.
+     * @param direction desired media direction:
+     *                       'sendrecv', 'sendonly', 'recvonly', 'inactive'
+     */
+    @SuppressWarnings("unused") // Used by plugins (Yuri)
```

Comment:
Do we actually still have plugins?

Reply:
Not sure.  @mstyura are you still using this?

Reply:
@bbaldino thanks for asking! I'm not really sure which version used in project I've worked on is such fresh. There will be some work to update, but I'm curios is there alternative way to tell bridge to not to route media to endpoint? Use case: endpoint is put on hold due to concurrent call, so not media need to be routed to it to save some bandwidth.

Reply:
I don't think we have any other method for it.  I'm fine with keeping it.